modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemsList.js

```diff
@@ -28,12 +29,14 @@ function CartItemsList() {
 		summaryDataMapper,
 	} = useContext(MiniCartContext);
 
-	const {cartItems = [], summary = {}} = cartState;
+	const {cartItems = [], id, summary = {}} = cartState;
 
 	return (
 		<div className="mini-cart-items-list">
 			<CartViews.ItemsListActions />
 
+			{cartState.accountId && id && <CartQuickAdd />}
```

Comment:
If a cart exists it will always be tied to a user/account and the cart itself will have an id. I haven't specifically checked this, but I'll add the `!!` just to be sure as this is a line that basically does the entire permission check for the component. Good catch